tests/_pure_prompt_beginning.test.fish

```diff
@@ -1,13 +1,5 @@
 source $current_dirname/../functions/_pure_prompt_beginning.fish
 
-@test "_pure_prompt_beginning: print prompt with newline for existing session" (
-    set _pure_fresh_session false
-
-    _pure_prompt_beginning
-) = '\n\r\033[K'
-
-@test "_pure_prompt_beginning: print prompt without newline for new session" (
-    set _pure_fresh_session true
-
+@test "_pure_prompt_beginning: clear line before printing prompt" (
```

Comment:
Why remove the tests for what is now `_pure_prompt_new_line`?

Reply:
Tests are failing for some reason. I don't know why.

Reply:
Probably because the function is registered as an event handler

Reply:
Yeah that's something I was wondering about, I'll check it out when I'm back home